RSAKeyVaultProvider/RSAKeyVaultProvider.csproj

```diff
@@ -18,9 +18,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Nerdbank.GitVersioning" Version="2.0.37-beta" PrivateAssets="all" />
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="2.3.2" />
+    <PackageReference Include="Nerdbank.GitVersioning" Version="2.0.37-beta" PrivateAssets="all" />
     <PackageReference Include="SourceLink.Create.CommandLine" Version="2.1.2" PrivateAssets="all" />
+    <PackageReference Include="System.Net.Http" Version="4.3.2" />
```

Comment:
Not ideal, but doing that was necessary to fix the infamous `Inheritance security rules violated by type: 'System.Net.Http.WebRequestHandler'. Derived types must either match the security accessibility of the base type or be less accessible.` error you reported last year.

Lemme know if you prefer pinning the version in the tests project instead of the main project.